### DISCUSSION TITLE
How can I use this plugin in VueJS or NuxtJS projects?

### DISCUSSION THREAD

**Discussion by hasib-devs** [xmppjs/xmpp.js#964] [Q&A]
I'm new to this plugin, is there any good article or video about using this plugin in React, Vue front-end library?

**Comment by japodhidev** [comment]
I hope this helps. I resorted to using a CDN as documented in this [README](https://github.com/xmppjs/xmpp.js/blob/main/packages/client/README.md). I'm currently working on Nuxt v3 & using `v0.13.1` of this library. To include the script tag from `https://unpkg.com/@xmpp/client@0.13.1/dist/xmpp.min.js` for example, you can use the `useHead`  composable. Here's a code snippet:

```js
useHead({
  title: 'Chat Page',
  script: [
    {
      src: 'https://unpkg.com/@xmpp/client@0.13.1/dist/xmpp.min.js',
      body: true
    }
  ]
});
```

You can access the client through the `window.XMPP.client` object once the script has been loaded

All the best.
